.github/workflows/build.yml

```diff
@@ -25,24 +25,29 @@ jobs:
           server-username: MAVEN_USERNAME
           server-password: MAVEN_CENTRAL_TOKEN
 
-      - name: Build project
-        if: runner.os != 'Linux'
+      - name: Build project (macOS)
+        if: runner.os == 'macOS'
         run: |
-          mvn -B -ntp verify -f rta
+          ./mvnw -B -ntp verify -f rta
+
+      - name: Build project (Windows)
+        if: runner.os == 'Windows'
+        run: |
+          .\mvnw -B -ntp verify -f rta
```

Comment:
does this needs to be `.\mvnw.cmd` ?

Reply:
it works fine as is (I tested it and also job is green), it runs on powerShell, so .cmd is the default extension.